src/main/java/seedu/address/model/statistics/StatisticData.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.statistics;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import javafx.scene.chart.PieChart.Data;
+
+/**
+ * Represents a data point in the Statistical Data.
```

Comment:
I'm not too sure this means. Did you mean it is a data point in the pie chart? 
```suggestion
 * Represents a data point in the Pie Chart.
```

Reply:
mmmmm sorry I realised this javadoc was back before we changed it to only show pie chart stats, thats why I used a more general stats data term as it was supposed to be able to change to data for different charts. Will change now thx for the catch!